README.md

```diff
@@ -63,6 +63,7 @@ Prerequisites:
 
 ```bash
 git clone git@github.com:palantir/tslint.git --config core.autocrlf=input --config core.eol=lf
+cd tslint
```

Comment:
Haha good catch! Would you mind sending this in as a separate PR? That way it's not blocked on requested changes in this PR.